drv/PriorityQueue.drv

```diff
@@ -60,6 +60,23 @@ public interface PRIORITY_QUEUE extends PriorityQueue<KEY_CLASS> {
 
 	default KEY_GENERIC_TYPE LAST() { throw new UnsupportedOperationException(); }
 
+	/** Returns the index element of the queue.
+	 * 
+ 	 * <p>This default implementation just throws an {@link UnsupportedOperationException}.
+ 	 * @see #peek()
+	 * @param index of the element.
+	 * @return the indexed element.
+	 * @throws NoSuchElementException if the index is not in the range
+	 */
+
+	default KEY_GENERIC_TYPE PEEK(int index) { throw new UnsupportedOperationException(); }
+
+	boolean REMOVE_KEY(KEY_GENERIC_TYPE x);
```

Comment:
Add documentation here. This removes the *first occurence* of `x`, not all.

Reply:
Ok, wasn't sure what doc to add there. Thanks for that help